src/main/java/org/twilmes/sql/gremlin/adapter/converter/schema/calcite/GremlinSchema.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.twilmes.sql.gremlin.adapter.converter.schema.calcite;
+
+import com.google.common.collect.ImmutableMap;
+import lombok.AllArgsConstructor;
+import org.apache.calcite.schema.Table;
+import org.apache.calcite.schema.impl.AbstractSchema;
+import org.twilmes.sql.gremlin.adapter.converter.schema.gremlin.GremlinEdgeTable;
+import org.twilmes.sql.gremlin.adapter.converter.schema.gremlin.GremlinTableBase;
+import org.twilmes.sql.gremlin.adapter.converter.schema.gremlin.GremlinVertexTable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Created by twilmes on 9/22/15.
+ * Modified by lyndonb-bq on 05/17/21.
```

Comment:
I see you've edited the message in the other file, does it need to be updated here too?